doc/glossary.rst

```diff
@@ -0,0 +1,257 @@
+
+.. currentmodule:: nilearn
+
+The Glossary provides short definitions of neuro-imaging concepts as well
+as Nilearn specific vocabulary. 
+
+If you wish to add a missing term, please `create a new issue`_ or
+`open a Pull Request`_.
+
+.. glossary::
+    :sorted:
+
+    
+    ANOVA
+    Anova
+        Analysis of variance.
+
+    AUC
+        Area under the curve.
+
+    BIDS
+    bids
+        `Brain Imaging Data Structure`_ is a simple and easy to adopt way
+        of organizing neuroimaging and behavioral data.
+    
+    BOLD
+    Bold
+    bold
+        Blood oxygenation level dependent.
+
+    CanICA
+        Canonical independant component analysis.
+
+    contrast
+    contrasts
+        A `contrast`_ is a linear combination of variables (parameters or
+        statistics) whose coefficients add up to zero, allowing comparison
+        of different treatments.
+
+    Decoding
+        `Decoding`_ consists in predicting, from brain images, the conditions
+        associated to trial.
+
+    EEG
+        Electroencephalography.
+    
+    EPI
+        Echo-Planar Imaging.
+
+    FDR correction
+        `False discovery rate`_ controlling procedures are designed to control
+        the expected proportion of "discoveries" (rejected null hypotheses)
+        that are false (incorrect rejections of the null).
+
+    FIR
+        Finite impulse response.
+
+    fMRI
+    fmri
+        Functional magnetic resonance imaging is based on the fact that 
+        when local neural activity increases, increases in metabolism and
+        blood flow lead to fluctuations of the relative concentrations of
+        oxyhaemoglobin (the red cells in the blood that carry oxygen) and
+        deoxyhaemoglobin (the same red cells after they have delivered the
+        oxygen). Oxyhaemoglobin and deoxyhaemoglobin have different magnetic
+        properties (diamagnetic and paramagnetic, respectively), and they
+        affect the local magnetic field in different ways.
+        The signal picked up by the MRI scanner is sensitive to these
+        modifications of the local magnetic field.
+
+    FPR correction
+    FPR
+    fpr
+        False positive rate correction.
+
+    FREM
+        `FREM`_ means "Fast ensembling of REgularized Models". It uses an implicit
+        spatial regularization through fast clustering and aggregates a high 
+        number of estimators trained on various splits of the training set, thus 
+        returning a very robust decoder at a lower computational cost than other 
+        spatially regularized methods. 
+
+    functional connectivity
+    functional connectome
+        A `functional connectome`_ is a set of connections representing brain 
+        interactions between regions.
+
+    FWER correction
+        `Family-wise error rate`_ is the probability of making one or more
+        false discoveries, or type I errors when performing multiple
+        hypotheses tests.
+
+    GLM
+        General Linear Model.
```

Comment:
This is the name of the models traditionally fit to fMRI data, where one linear model is fit to each voxel time course.